docs/source/caching/normalized-cache.mdx

```diff
@@ -172,31 +176,29 @@ val response = apolloClient.query(query)
 
   // Always use the network and never check the cache, even if network fails
   .fetchPolicy(FetchPolicy.NetworkOnly)
-
-  // Execute the query
-  .execute()
-```
-The `CacheAndNetwork` policy can emit multiple values, so you call `toFlow()` instead of `execute()`:
-
-```kotlin
-apolloClient.query(query)
-
-  // Check the cache and also use the network (1 or 2 values can be emitted)
+  
+  // Check the cache and also use the network
   .fetchPolicy(FetchPolicy.CacheAndNetwork)
 
   // Execute the query and collect the responses
   .toFlow().collect { response ->
-      // ...
+    // ...
   }
 ```
+Note that cache misses **will** emit a response with a non null `.exception`, meaning that some of these policies can emit multiple values:
+- `CacheFirst` can emit 1 or 2 values
+- `NetworkFirst` can emit 1 or 2 values
+- `CacheAndNetwork` will emit 2 values
+
+For that reason you should use `.toFlow()` rather than `.execute()` when using these policies.
```

Comment:
Mixed feelings about this. `.execute()` works well for `CacheFirst` and `NetworkFirst` and AFAIK, it's one of the main usages today. 
`CacheAndNetwork` is different in that `execute()` doesn't work at all. Ultimately, we might want to deprecate `execute()` altogether but until we do, I wouldn't recommend against it for `CacheFirst` and `NetworkFirst`

Reply:
Totally right! Replacing with:

> Note: `.execute()` filters out cache or network errors to return a single success response, however since `CacheAndNetwork` can emit 2 success responses, you should use `.toFlow()` when using that policy.